src/didstore.c

```diff
@@ -330,11 +347,27 @@ static int load_credmeta(DIDStore *store, CredentialMetaData *meta, const char *
 
     rc = CredentialMetaData_FromJson(meta, data);
     free((void*)data);
-    if (rc)
-        return rc;
+    if (rc < 0)
+        return -1;
 
+    //set last modified
+    if (get_file(path, 0, 6, store->root, DID_DIR, did, CREDENTIALS_DIR,
+            fragment, CREDENTIAL_FILE) == -1) {
+        CredentialMetaData_Free(meta);
```

Comment:
why need to call CredentialMetaData_Free() to free meta, which is feeded by caller ?